.github/workflows/dev.yml

```diff
@@ -0,0 +1,58 @@
+#  Name of workflow
+name: Dev workflow
+
+# When workflow is triggered
+on:
+  workflow_dispatch:
+
+# Jobs to carry out
+jobs:
+  build:
+    # Operating system to run job on
+    runs-on: ubuntu-latest
+    # Environment to run job on
+    environment: staging
+    # Environment variables
+    env:
+      DRUPAL_CONCURRENT_FILE_REQUESTS: 20
+      DRUPAL_REQUEST_TIMEOUT: 6000000
+      DRUPAL_URL: 'https://cms.dev.lib.umich.edu/'
```

Comment:
This can be changed to a github variable within the staging environment. (Github Actions variables didn't exist when I wrote up the other actions)

Reply:
we used to be able to use those in netlify. getting them back would be great

Reply:
It's a thing in github actions now.

Reply:
I added a variable for staging. to use it, replace the url for the cms with `${{ vars.DRUPAL_URL}}`

You should be able to look at it if you go to settings --> environments --> staging